Assets/MRTK/Core/Providers/Hands/BaseHandVisualizer.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
```

Comment:
nit: I'm not sure I see anything being added to this script that would require a new `using`. Am I overlooking something?

Reply:
removed!